blosc/schunk.c

```diff
@@ -146,7 +190,11 @@ int blosc2_schunk_append_chunk(blosc2_schunk *schunk, uint8_t *chunk, bool copy)
   int32_t nbytes = sw32_(chunk + 4);
   int32_t cbytes = sw32_(chunk + 12);
 
-  if ((schunk->nchunks > 0) && (nbytes > schunk->chunksize)) {
+  if (schunk->chunksize == -1) {
+    schunk->chunksize = nbytes;  // Only update chunksize when it is the first chunk
```

Comment:
The comment is not valid anymore.  I suggest to replace it by: "the super-chunk is initialized now"